qiita_db/support_files/patches/python_patches/30.py

```diff
@@ -0,0 +1,75 @@
+from qiita_db.sql_connection import TRN
+from qiita_db.metadata_template.constants import (NA_VALUES, TRUE_VALUES,
+                                                  FALSE_VALUES)
+from qiita_db.metadata_template.prep_template import PrepTemplate
```

Comment:
`from qiita_db.metadata_template import SampleTemplate, PrepTemplate` and remove line 5